frontend/src/modules/app/layouts/NoLayoutRoute.tsx

```diff
@@ -1,11 +1,17 @@
 /**
- * This is the doc comment for Empty Layout
- * @packageDocumentation
- * @module empty-page-layout
+ * @summary Layout with full empty container
```

Comment:
What is a "full empty container"? Where is `NoLayoutRoute` useful, or when would it be useful?

Reply:
Full Empty container resembles Container with no default layout for Sidebar, Toolbar, Footer, etc. Its just plain Empty container. It is not in use anywhere as of now but can be used if any such page is required, if not can be marked as deprecated